pkg/loki/modules.go

```diff
@@ -140,18 +137,9 @@ func (t *Loki) initServer() (services.Service, error) {
 		})
 	}(t.Server.HTTPServer.Handler)
 
-	// Allow us to receive http/2 cleartext in addition to http/1.1. This needs to be
-	// the outermost handler so it can upgrade requests if necessary
-	t.Server.HTTPServer.Handler = http2CleartextHandler(t.Server.HTTPServer.Handler)
-
```

Comment:
Isn't this piece remove http2 from all our handlers? Do we want this (cc @slim-bean )?

Reply:
Good catch. :+1:

Reply:
http2 was only added for this feature so removing it would just be putting us back where we were